### PR TITLE
Feature/invalid status

### PR DESCRIPTION
**Pull request by GautierBlandin**
Feature feed PR for the SDK

### CODE REVIEW

File: src/services/feed.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import { newFeedElements } from '../constructor/feed';
+
+export class FeedService implements Service {
+    readonly subpath = '/feed';
+
+    /**
+     * Remote service status
+     * @return whether the service is alive or not
+     */
+    public async alive(): Promise<Status.ServiceStatus> {
```
Comment (ArthurKnoep) [inline src/services/feed.ts:17]: Waiting for the addition of a healthcheck method

File: src/services/feed.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        } catch (err) {
+            throw dispatchError(err);
+        }
+    }
+
+    /**
+     * Retrieves the feeds elements of a user
+     * @return A promise of the result of the feed elements retrieving transaction
+     */
+    public async retrieveAllFeedElements(startAt: Date, seen?: boolean, type?: string, pagination?: Pagination): Promise<Feed.PaginatedFeedElements> {
```
Comment (ArthurKnoep) [inline src/services/feed.ts:31]: `startAt` should not be required imo

**Review by ArthurKnoep — CHANGES_REQUESTED** [review]
Ok, just waiting some changes on acropolis

File: src/services/feed.ts (hunk 1 above)
Comment (GautierBlandin) [reply src/services/feed.ts:18]: I think it should be good now ?

**Review by ArthurKnoep — APPROVED** [review]
Seems OK, you have a conflict on the package version tho

**Pull request merged**